src/Runner.Common/JobServerQueue.cs

```diff
@@ -804,6 +833,34 @@ private async Task UploadSummaryFile(SummaryUploadFileInfo file)
                 }
             }
         }
+
+        private async Task UploadResultsStepLogFile(ResultsUploadFileInfo file)
+        {
+            bool uploadSucceed = false;
+            try
+            {
+                Trace.Info($"Starting upload of step log file to results service {file.Name}, {file.Path}");
+                var cancellationTokenSource = new CancellationTokenSource();
```

Comment:
why create a cancellationTokenSouce here?
i didn't find anything that calls `Cancel()` on it.

Reply:
I think this was created because `RawHttpClientBase` expected one.

I am making a lot of changes in the http client in a separate branch and will submit a new PR very soon, and I will remove those `cancellationTokenSource` there.

Reply:
you can just pass `CancellationToken.None` to the `await _jobServer.CreateResultsStepLogAsync()` if you don't want to have cancellation for now.